### PR TITLE
Issue #8151: Update XpathRegressionAvoidNoArgumentSuperConstructorCallCheckTest to have at least 2 tests

### PR DESCRIPTION
**Pull request by Gaurav-Punjabi**
Partially resolves issue #8151

### CODE REVIEW

**Comment by Gaurav-Punjabi** [comment]
@strkkk @romani Please review

File: src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionAvoidNoArgumentSuperConstructorCallTest.java [hunk 1]
```diff
@@ -62,4 +62,30 @@ public void testOne() throws Exception {
...
+
+        final DefaultConfiguration moduleConfig =
+                createModuleConfig(CLASS);
+
+        final String[] expectedViolation = {
+            "5:9: " + getCheckMessage(CLASS,
+                AvoidNoArgumentSuperConstructorCallCheck.MSG_CTOR),
+        };
+
+        final List<String> expectedXpathQueries = Collections.singletonList(
```
Comment (rdiachenko) [inline src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionAvoidNoArgumentSuperConstructorCallTest.java:80]: This is the same path as in the first test, thus the same test case. Could you create and test a more complex case like this one:
```
public class A {
  void fun() {
    class B {
      B() {
        super(); // warn
      }
    }
  }
}
```

Comment (Gaurav-Punjabi) [reply src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionAvoidNoArgumentSuperConstructorCallTest.java:80]: Sure

Comment (Gaurav-Punjabi) [reply src/it/java/org/checkstyle/suppressionxpathfilter/XpathRegressionAvoidNoArgumentSuperConstructorCallTest.java:80]: Done, please check

**Comment by romani** [comment]
@Gaurav-Punjabi , please rebase , I not surer why appveyor is failed to run.

**Comment by Gaurav-Punjabi** [comment]
> @Gaurav-Punjabi , please rebase , I not sure why appveyor is failed to run.

Done 👍

**Pull request merged**